### ISSUE TITLE
Accessibility: Obtain Section 508 Compliance Certification for manage.get.gov

### ISSUE THREAD

**Issue by PaulKuykendall** [cisagov/getgov#2577]
### Issue description

As part of the quarterly compliance testing per the [.gov Accessibility Plan](https://docs.google.com/document/d/1Rt6XN7yZKvdA3A2KRkv54SzmkjQ4jOb6Xknhd0mOvKE/edit?pli=1), request certification from OAST, remediate as neccessary, to obtain an updated Compliance Determination Form (CDF) indicating successful compliance.

https://manage.get.gov/

### Acceptance criteria

- [x] Request Submitted to OAST and ServiceNow ticket number linked here.
- [x] cisagov/manage.get.gov#2835 
- [ ] CDF has been received from OAST, indicating satisfactory compliance status.
- [ ] CDF has been saved in program compliance documentation folder.
- [ ] Update Accessibility Compliance plan, as necessary and appropriate.

### Additional context

OAST Ticket number: ICTR0044555

### Links to other issues

* cisagov/get.gov#356 (get;gov)
* cisagov/manage.get.gov#2578 (admin) 

Blocked by:
- cisagov/manage.get.gov#2835 

### Prioritization Comments
**WCAG Conformance Level:** 
**WCAG Standard that applies:** 
**Critical Failure if Not Met?:**  
**Prioritization:** 
**Reason:**

**Comment by PaulKuykendall** [comment]
Submitted remediation plan and will request re-test and CDF. (Discussing with Matti/Aaron this week)
